docs/_providers/hostingde.md

```diff
@@ -0,0 +1,39 @@
+---
+name: hosting.de
+title: hosting.de Provider
+layout: default
+jsId: hostingde
+---
+# hosting.de Provider
+
```

Comment:
Suggestion: Add a paragraph saying that this provider suports the API used by Hosting.de and  http.net, which both use the same API. (or whatever)

Reply:
I do not think that this is neccessary. http.net only provides registration services for bigger companies. Also http.net in mentioned later in the documentation.